sample-app/src/main/java/com/sinch/sample/conversation/contact/Update.java

```diff
@@ -0,0 +1,66 @@
+package com.sinch.sample.conversation.contact;
+
+import com.sinch.sample.BaseApplication;
+import com.sinch.sdk.domains.conversation.models.ChannelIdentity;
+import com.sinch.sdk.domains.conversation.models.ChannelType;
+import com.sinch.sdk.domains.conversation.models.Contact;
+import com.sinch.sdk.domains.conversation.models.LanguageType;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.logging.Logger;
+
+public class Update extends BaseApplication {
+
+  private static final Logger LOGGER = Logger.getLogger(Update.class.getName());
+
+  public Update() throws IOException {}
+
+  public static void main(String[] args) {
+    try {
+      new Update().run();
+    } catch (Exception e) {
+      LOGGER.severe(e.getMessage());
+      e.printStackTrace();
+    }
+  }
+
+  public void run() {
+
+    LOGGER.info("Update contact '%s'".formatted(conversationContactId));
+
+    var contact =
+        Contact.builder()
+            .setId(conversationContactId)
+            .setChannelIdentities(
+                Arrays.asList(
+                    ChannelIdentity.builder()
+                        .setChannel(ChannelType.MESSENGER)
+                        .setIdentity("+33123456789")
```

Comment:
Should not be a phone number for messenger but the messenger ID (same for instagram, and maybe also for kakao talk TBC)

Reply:
Modified sample

Reply:
Maybe your forgot a commit somewhere, I still see the phone number in the example 😞